Environment/Environment.pillar

```diff
@@ -806,24 +764,22 @@ right-clicking on the stack frames provide equivalent functionality.
 !!Finding methods
 @sec:methodFinder
 
-There are two tools in Pharo to help you find messages. They differ in both
-interface and functionality.
+There is one tool in Pharo to help you find messages. 
```

Comment:
I'm not sure we should phrase it explicitly as 'there is one tool in Pharo to help you find messages'. There's the Finder, also Spotter, plus I'm sure there's other methods.